numbers_in_words.gemspec

```diff
@@ -9,13 +9,14 @@ Gem::Specification.new do |gem|
   gem.summary       = "Example: 123.in_words # => \"one hundred and twenty three\", \"seventy-five point eight\".in_numbers # = > 75.8"
 
   gem.version       = NumbersInWords::VERSION
-  gem.authors       = ["Mark Burns"]
-  gem.email         = ["markthedeveloper@gmail.com"]
-  gem.homepage      = "http://github.com/markburns/numbers_in_words"
+  gem.authors       = ["Mark Burns", "Dimid Duchovny"]
+  gem.email         = ["dimidd@gmail.com"]
```

Comment:
You can add your email, just don't replace mine, also leave the homepage as it is